plugins/DiskSize/plugin.rb

```diff
@@ -63,9 +63,7 @@ def report(fqdn = '*', days = 1)
     # format and make json/csv thing
     dataset.all.to_json
   end
+
+  DiskSize.new.register
```

Comment:
I think the register method should be called initialize if we want it be called on first run or just removed completely